packages/cli/src/utils/checks.ts

```diff
@@ -323,6 +323,9 @@ class CheckBuilder {
       `Account isn't a member of a validator group`,
       this.withValidators(async (v, _signer, account) => {
         const { affiliation } = await v.getValidator(account)
+        if (eqAddress(affiliation!, NULL_ADDRESS)) {
```

Comment:
What is the result if affiliation is `null`? Based on some of the code in the Validator contracts wrapper, the following seems reasonable.
```suggestion
        if (!affiliation || eqAddress(affiliation, NULL_ADDRESS)) {
```